disk/datadog_checks/disk/disk.py

```diff
@@ -6,6 +6,7 @@
 import os
 import platform
 import re
+import xml.etree.ElementTree
```

Comment:
`import xml.etree.ElementTree` will only import `xml`.

You can instead use `import xml.etree.ElementTree as ET` and then use `ET.fromstring(line)` as suggested in the doc.